tests/helpers.lua

```diff
@@ -29,12 +29,13 @@ function M.session_setup_luasnip(opts)
 	local no_snip_globals = opts.no_snip_globals ~= nil and opts.no_snip_globals
 		or false
 
+	-- stylua: ignore
 	helpers.exec("set rtp+=" .. os.getenv("LUASNIP_SOURCE"))
 	helpers.exec(
-		string.format(
-			"source %s/plugin/luasnip.vim",
-			os.getenv("LUASNIP_SOURCE")
-		)
+		("source %s/plugin/luasnip.vim"):format(os.getenv("LUASNIP_SOURCE"))
+	)
+	helpers.exec(
+		("luafile %s/plugin/luasnip.lua"):format(os.getenv("LUASNIP_SOURCE"))
```

Comment:
You don't need to change it, but it's also valid to use `source` here.

Reply:
The rest seems "good to go".

Reply:
Ah, right!
Yeah, I'll leave it like this :+1: